ra_aid/model_detection.py

```diff
@@ -86,6 +109,10 @@ def should_use_react_agent(model: BaseChatModel) -> bool:
     model_name = get_model_name_from_chat_model(model)
     normalized_model_name = normalize_model_name(model_name)
 
+    if is_deepseek_v3(model_name) and is_deepseek_model(model):
+        logger.debug(f"Using create_react_agent for DeepSeek model: {model_name}")
+        return True
+
```

Comment:
We should be getting away from special case logic. The default agent backend for the model should be specified in `models_params.py`.